server/src/main/java/io/spine/examples/shareaware/server/watchlist/UserWatchlistsProjection.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.shareaware.server.watchlist;
+
+import io.spine.core.Subscribe;
+import io.spine.core.UserId;
+import io.spine.examples.shareaware.watchlist.UserWatchlists;
+import io.spine.examples.shareaware.watchlist.event.WatchlistCreated;
+import io.spine.server.projection.Projection;
+
+import static io.spine.examples.shareaware.watchlist.UserWatchlists.*;
+
+/**
+ * Builds display information for all user's watchlists.
+ */
+final class UserWatchlistsProjection
+        extends Projection<UserId, UserWatchlists, UserWatchlists.Builder> {
+
+    @Subscribe
+    void on(WatchlistCreated e) {
+
+        WatchlistView watchlist = WatchlistView.newBuilder()
+                                                   .setId(e.getWatchlist())
```

Comment:
Please adjust the formatting rules in IDEA according to the IDEA settings residing in [`SpineEventEngine/config` repository](https://github.com/SpineEventEngine/config/tree/master/.idea).

Ideally, we want to have something like this (an excerpt from `1.x-dev` branch of `SpineEventEngine/core-java`):

```java
            DispatchOutcome outcome = DispatchOutcome
                    .newBuilder()
                    .setPropagatedSignal(event.messageId())
                    .setInterrupted(interruption)
                    .vBuild();
```
@Vladyslav-Kuksiuk FYI.